clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -13,6 +13,7 @@
   {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
   {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
   {{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+  {{$USE_CSI := DefaultParam .USE_CSI false}}
```

Comment:
should this param be more specific like `DEPLOY_PD_CSI_DRIVER` or something?

Reply:
I think in the future, we may want to support more drivers to test, so its vague here in the config on purpose.

Reply:
USE_CSI sounds fine - (in test we're not deploying anything, just potentially using the functionality)